### DISCUSSION TITLE
LookupEditor(inplaceAdd) in conjunction with customLookupEditor.ts

### DISCUSSION THREAD

**Discussion by kilroyFR** [serenity-is/Serenity#5836] [Help] [answered]
hey there, i am using a custom MyRuleLookup where i mainly customize PrepareQuery in order to Sort displayed fields.
So far it works great.

I have also created a MyRuleLookupEditor.ts where i customize mainly the multiple fields displayed in getItemText().

My problem is that i would like to use these 2 (i mean sorted lookup + custom display) on my xxxRow.cs and enabling InPlaceAdd

`[LookupEditor(typeof(MyRuleLookup), InplaceAdd = true)]` 
>> works but i cannot see the multiple fields formatted in the editor (it only displays the Label of the lookup)

`[RopThirdPartyRulesLookupEditor]`
>> works and displays the multiple fields as set in get_item_text() but i cannot add InPlace addition

I guess i am not the only one in need of this behavior ?

Thanks for any help

**Comment by kilroyFR** [comment, marked as answer]
I guess the answer is here https://github.com/serenity-is/Serenity/issues/3559

But would love to get help on this

**Reply by kilroyFR** [reply]
Found it adding a reference to <row, LookupEditorOptions> in my Editor Constructor.
` [RopThirdPartyRulesLookupEditor(InPlaceAdd = true]` is now available

**Reply by axelcasauran** [reply]
where should I add the "<row, LookupEditorOptions" ?
